app/src/main/java/de/tum/in/tumcampusapp/activities/BarrierFreeInfoActivity.java

```diff
@@ -0,0 +1,51 @@
+package de.tum.in.tumcampusapp.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.activities.generic.BaseActivity;
+
+
+public class BarrierFreeInfoActivity extends BaseActivity {
+
+    private Button button1;
+
+    public BarrierFreeInfoActivity(){
+        super(R.layout.activity_barrier_free_info);
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        ListView listView = (ListView) findViewById(R.id.activity_barrier_free_list_view);
+
+
+        String[] values = new String[]{
+                "Contact",
+                "Nearby Facilities",
+                "General Information"
```

Comment:
does this always contain only there three items? what about moving this to the layout directly or at least make the array static final so that it is instantiated only once